### PR TITLE
Remove Resource time line from cephblockpool performance card

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
Screenshot from 2023-02-17 22-42-35

**Comment by TimothyAsirJeyasing** [comment]
@SanjalKatiyar Please review

File: packages/ocs/dashboards/block-pool/utilization-card.tsx [hunk 1]
```diff
@@ -29,7 +27,7 @@ export const UtilizationCard: React.FC = () => {
           <UtilizationDurationDropdown />
         </CardActions>
       </CardHeader>
-      <UtilizationBody>
```
Comment (GowthamShanmugam) [inline packages/ocs/dashboards/block-pool/utilization-card.tsx:32]: Why we are removing this? what is the bug?

Comment (TimothyAsirJeyasing) [reply packages/ocs/dashboards/block-pool/utilization-card.tsx:32]: This UtilizationBody adds some time line details which is not very meaningful for this graph

Comment (SanjalKatiyar) [reply packages/ocs/dashboards/block-pool/utilization-card.tsx:32]: add "before" and "after" screenshots both...

Comment (SanjalKatiyar) [reply packages/ocs/dashboards/block-pool/utilization-card.tsx:32]: so that it is clear to anyone who is not aware of this bug...

Comment (TimothyAsirJeyasing) [reply packages/ocs/dashboards/block-pool/utilization-card.tsx:32]: Sure will do it

**Comment by TimothyAsirJeyasing** [comment]
Before:
Screenshot from 2023-02-20 15-23-31


After:
Screenshot from 2023-02-17 22-42-35

**Review by GowthamShanmugam — APPROVED** [review]

**Pull request merged**